<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_31" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_23">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="widget_33" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_24">
         <property name="bottomMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Informazioni personali</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_40" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label_31">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Matricola:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="matricola">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_38" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Nome: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nome">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_39" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cognome:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cognome">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_41" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Data di nascita:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataNascita">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_50" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <property name="topMargin">
          <number>35</number>
         </property>
         <item>
          <widget class="QPushButton" name="modificaProfiloButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#modificaProfiloButton {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#modificaProfiloButton:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
           </property>
           <property name="text">
            <string>Modifica dati personali</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Amministratore/Resources/edit icon.png</normaloff>../../Amministratore/Resources/edit icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modificaPasswordButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#modificaPasswordButton {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#modificaPasswordButton:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
           </property>
           <property name="text">
            <string>Cambia password</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Amministratore/Resources/edit icon.png</normaloff>../../Amministratore/Resources/edit icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="eliminaAccountButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#eliminaAccountButton {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#eliminaAccountButton:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
           </property>
           <property name="text">
            <string>Elimina account</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Amministratore/Resources/delete-icon.png</normaloff>../../Amministratore/Resources/delete-icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_35" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_29">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_43" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_31">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="label_35">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Prestiti</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="prestitoLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(221, 0, 0);</string>
           </property>
           <property name="text">
            <string>Non sono presenti prestiti</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="prestito1Widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="prestito1">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Prestito 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="visualizzaPrestito1Button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#visualizzaPrestito1Button {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#visualizzaPrestito1Button:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Visualizza</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../Amministratore/Resources/eye icon.jpg</normaloff>../../Amministratore/Resources/eye icon.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="prorogaPrestito1Button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#prorogaPrestito1Button {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#prorogaPrestito1Button:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Proroga</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="prestito2Widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="prestito2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Prestito 2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="visualizzaPrestito2Button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#visualizzaPrestito2Button {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#visualizzaPrestito2Button:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Visualizza</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../Amministratore/Resources/eye icon.jpg</normaloff>../../Amministratore/Resources/eye icon.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="prorogaPrestito2Button">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#prorogaPrestito2Button {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#prorogaPrestito2Button:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Proroga</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_36" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_36">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_64" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_37">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="label_52">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Prenotazioni</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="prenotazioneLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(221, 0, 0);</string>
           </property>
           <property name="text">
            <string>Non è presente alcuna prenotazione</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="prenotazioniWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="prenotazione">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Prenotazione</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="visualizzaPrenotazioneButton">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#visualizzaPrenotazioneButton {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#visualizzaPrenotazioneButton:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Visualizza</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../Amministratore/Resources/eye icon.jpg</normaloff>../../Amministratore/Resources/eye icon.jpg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="annullaPrenotazioneButton">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#annullaPrenotazioneButton {
	border: 1px solid;
	background-color:rgb(230, 230, 230);
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}
QPushButton#annullaPrenotazioneButton:hover {
	
	background-color: rgb(207, 207, 207);
}</string>
              </property>
              <property name="text">
               <string>Annulla</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../Amministratore/Resources/red x icon.png</normaloff>../../Amministratore/Resources/red x icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
